static/css/jira-manual-app-creation.css

```diff
@@ -0,0 +1,105 @@
+.jiraManualAppCreation {
+  max-width: 560px;
+  margin: 0 auto;
+}
+
+h1 {
+  text-align: center;
+  margin: 1.75em 0 2.5em;
+  font-size: 1.75rem;
+  font-weight: 600;
+}
+
+.jiraManualAppCreation__formTitle {
+  font-size: 1.2rem;
+  font-weight: 600;
+  margin: 1.25em 0 1em;
+  display: flex;
+  align-items: center;
+}
+
+.jiraManualAppCreation__formTitle > .aui-icon {
+  margin-left: 4px;
+}
+
+.jiraManualAppCreation__formSubTitle {
+  font-size: 1rem;
+  font-weight: 400;
+  margin: 1.25em 0;
+  color: #344563;
+}
+
+.field-group {
+  margin-bottom: 1.1em !important;
```

Comment:
Do we need !important here? Can we apply this style with specificity?

Reply:
Replaced important using specificity